packages/components/stories/HeaderBar.js

```diff
@@ -140,11 +143,13 @@ const props = {
 
 const decoratedStories = storiesOf('HeaderBar', module)
 	.addDecorator(story => (
-		<div>
-			{story()}
-			<div className="container" style={{ paddingTop: 40 }} />
-			<IconsProvider defaultIcons={icons} />
-		</div>
+		<I18nextProvider i18n={i18n}>
+			<div>
```

Comment:
Since there is a wrapper declared `TranslateWrapper.component.js` above, why explicitely declare `I18nextProvider` here